src/argilla/server/security/model.py

```diff
@@ -62,7 +62,7 @@ class UserCreate(BaseModel):
 class UserGetter(GetterDict):
     def get(self, key: str, default: Any = None) -> Any:
         if key == "full_name":
-            return f"{self._obj.first_name} {self._obj.last_name}"
+            return " ".join(filter(None, [self._obj.first_name, self._obj.last_name]))
```

Comment:
I feel too much logic here just to prevent default value. Wdyt @jfcalvo ?

```suggestion
            return f"{self._obj.first_name} {self._obj.last_name or ''}"
```

Reply:
I will apply this feedback in the base PR.

Reply:
👍